packages/dso-toolkit/src/global/spacing.scss

```diff
@@ -10,7 +10,8 @@
 // Original spacing of dso-button-row
 // ----------------------------------
 // To Do: re-evaluate vertical rhythm
-:where(.dso-button-row) {
+:where(.dso-button-row),
+:where(.dso-file-upload) {
```

Comment:
Waar is deze change voor?

Reply:
De `dso-file-upload` werd beïnvloed door de `modifier`. Hierdoor kreeg het styling mee, alleen wat `margin`. Omdat de `modifier` aangepast werd naar `mode` hebben we deze styling even zo ondergebracht.

Reply:
Dat is inderdaad wat wij zagen en waarom wij dit (toen) nodig hadden. Maar als ik het nu weer bekijk, zie ik geen verandering meer hiermee.